control/control.go

```diff
@@ -814,7 +848,7 @@ func (p *pluginControl) FetchMetrics(ns []string, version int) ([]core.Cataloged
 }
 
 func (p *pluginControl) GetMetric(ns []string, ver int) (core.CatalogedMetric, error) {
-	return p.metricCatalog.Get(ns, ver)
+	return p.metricCatalog.GetMetric(ns, ver)
 }
 
 func (p *pluginControl) GetMetricVersions(ns []string) ([]core.CatalogedMetric, error) {
```

Comment:
What is more, the dynamic metrics should be taken into consideration - that way the another command should be use.

Example:

Metric catalog:
/intel/procfs/processes/parked
/intel/procfs/processes/waking
/intel/procfs/processes/*/ps_vm
/intel/procfs/processes/*/ps_rss

`snapctl metric get /intel/procfs/processes/*/ps_vm` should return only information about this single metric